app/services/slack/argument_parser.rb

```diff
@@ -48,6 +49,7 @@ class RestOptions
         developer_get: %i[persisted_developer],
         developer_list: %i[],
         developer_delete: %i[persisted_developer],
+        developer_set: %i[],
```

Comment:
I think you can add `persisted_developer` developer again :)

Reply:
nice sure thing :D